Runbooks/startstop/1.0/AutoStop_CreateAlert_Parent.ps1

```diff
@@ -0,0 +1,277 @@
+﻿<#
+.SYNOPSIS
+ Runbook for shutdown the Azure VM based on CPU usage
+.DESCRIPTION
+ Runbook for shutdown the Azure VM based on CPU usage
+.EXAMPLE
+.\AutoStop_CreateAlert_Parent.ps1 -WhatIf $false -VMList "vm1,vm2"
+Version History
+v1.0   - Initial Release
+v2.0   - Added classic support
+#>
+
+Param(
+    [Parameter(Mandatory = $false, HelpMessage = 'Enter the value for WhatIf. Values can be either true or false')][bool]$WhatIf = $false,
+    [Parameter(Mandatory = $false, HelpMessage = 'Enter the VMs separated by comma(,) if you want to create alerts for VMs')][string]$VMList
+)
+
+function CheckValidAzureVM ($FilterVMList, $EnableClassicVMs) {
+    [boolean] $ISexists = $false
```

Comment:
Is this variable aptly named?

Reply:
Microsoft created this script. We can optimize and fix them in a different PR.

Reply:
:D